sklearn/tests/test_pipeline.py

```diff
@@ -949,29 +963,33 @@ def test_step_name_validation():
         # we validate in construction (despite scikit-learn convention)
         bad_steps3 = [('a', Mult(2)), (param, Mult(3))]
         for bad_steps, message in [
-            (bad_steps1, "Estimator names must not contain __: got ['a__q']"),
-            (bad_steps2, "Names provided are not unique: ['a', 'a']"),
-            (bad_steps3, "Estimator names conflict with constructor "
-                         "arguments: ['%s']" % param),
+            (bad_steps1, r"Estimator names must not contain __:"
+             r" got \['a__q'\]"),
```

Comment:
indentation is weird

also, not sure why you changed the strings?

Reply:
indentation is pep8, I think? I changed them because we need to use pytest raises now and that uses regexs.